lib/neverblock.rb

```diff
@@ -1,3 +1,5 @@
+require 'net/http'
```

Comment:
are v2.2 the same as master?

Reply:
v2.1 is the same as master -- v2.2 and v2.3 were fixes for CF-1931 that got reverted. Another solution was found in cloud_gateway. Think I'm going to make a another pull to revert those changes.